mix.exs

```diff
@@ -109,7 +109,7 @@ defmodule Wanda.MixProject do
       {:credentials_obfuscation, "3.0.0", override: true},
       {:jason, "~> 1.3"},
       {:yaml_elixir, "~> 2.9"},
-      {:trento_contracts, github: "trento-project/contracts", ref: "39abbf1", sparse: "elixir"},
+      {:trento_contracts, github: "trento-project/contracts", sparse: "elixir", ref: "1bff79a"},
```

Comment:
I guess this is temporal until we merge the contract PR, right?

Reply:
yes!

Reply:
Man, at the end you didn't wait until merging first the other to put here the merge commit XD
Now this commit doesn't even exist in the `main` branch, which looks difficult to debug